clash-prelude/src/Clash/Sized/Vector.hs

```diff
@@ -157,6 +144,7 @@ import Clash.XException
 >>> :set -XTypeApplications
 >>> :set -fplugin GHC.TypeLits.Normalise
 >>> import Clash.Prelude
+>>> import Data.Kind
```

Comment:
Maybe it's worth exporting this from `Clash.Prelude` as well, it's going to be more pervasive now we're moving away from `*`